src/ServicePulse.Host/app/modules/monitoring/js/directives/ui.particular.graph.js

```diff
@@ -75,17 +75,28 @@
                             .attr('stroke', 'gray');
 
                         var displayValue = lastValue.toFixed(2);
+                        var unit = "";
                         if (typeof scope.plotData.displayValue !== "undefined") {
-                            displayValue = scope.plotData.displayValue;
+                            displayValue = scope.plotData.displayValue.value;
+                            unit = scope.plotData.displayValue.unit;
                         }
 
                         chart.append("text")
```

Comment:
I know it's not part of your change, but can we switch to single quotes all over here to be consistent?